src/main/java/gregtech/api/unification/ore/OrePrefix.java

```diff
@@ -368,6 +368,12 @@
         pipeNormalRestrictive.addSecondaryMaterial(new MaterialStack(Materials.Iron, ring.materialAmount * 2));
         pipeLargeRestrictive.addSecondaryMaterial(new MaterialStack(Materials.Iron, ring.materialAmount * 2));
 
+        cableGtSingle.addSecondaryMaterial(new MaterialStack(Materials.Rubber, plate.materialAmount));
+        cableGtDouble.addSecondaryMaterial(new MaterialStack(Materials.Rubber, plate.materialAmount));
+        cableGtQuadruple.addSecondaryMaterial(new MaterialStack(Materials.Rubber, plate.materialAmount * 2));
+        cableGtOctal.addSecondaryMaterial(new MaterialStack(Materials.Rubber, plate.materialAmount * 3));
+        cableGtHex.addSecondaryMaterial(new MaterialStack(Materials.Rubber, plate.materialAmount * 4));
```

Comment:
These are not quite accurate to their current costs, but a PR reworking cable covering will come shortly after this (before release)

Reply:
Now that the PR to rework cable covering is in, are these values accurate to the current costs?

Reply:
The `cableGtHex` value is incorrect, which I will fix (should be 5)